samples/boot/oauth2resourceserver-opaque/src/main/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        opaque:
+          introspection-uri: ${mockwebserver.url}/introspect
```

Comment:
Just want to ensure we think about if/how this aligns with OAuth2 Log In. Do we support discovery? If an application is supporting oauth2 log in and resource server do we want them to have to configure the client id/secret twice?

Reply:
Yes, I think discovery makes sense through the `issuer-uri` once it supports the `/.well-known/oauth-authorization-server` endpoint, which defines `introspection_endpoint` as an attribute.

As for the client, ideally, the user wouldn't need to specify the same client twice as I imagine that the 95% case is that they are the same. If there is only one client in client registration, it seems reasonable to infer that client is the same for the introspection endpoint.